### ISSUE TITLE
How should the fix for the dependencies be handled?

### ISSUE THREAD

**Issue by Avey777** [RedashCommunity/redash#80]
I am installing the packages one by one in al_ds and found that the following packages will throw errors when run after installation.

```
# oauth2client==4.1.3 #bug
# dql==0.5.26 #bug
# snowflake-connector-python==3.0.2 #bug
# firebolt-sdk  #bug
```

I have not tested the following disabled dependency packages.
```
# pymapd==0.19.0
# ibm-db>=2.0.9

**Comment by Avey777** [comment]
@justinclift 
For these dependency packages, they should be tested one by one. If there are issues, they can either be fixed or disabled, but there needs to be progress in resolving them. I suggest that for those data sources that are no longer actively updated, if problems occur, they can be directly removed from support.

**Comment by justinclift** [comment]
Yeah, with the exception of the `oauth2client` package, I reckon disabling those other ones for now is probably the right move.

The `oauth2client` one will probably turn out to be important though, as OAuth is commonly used in web based authentication.  So there *could* be a bunch of people that need it.

That being said, I agree with you. Lets just remove all of these for now and see how the resulting preview image goes.

It seems like it *should* go ok, and we'll have a working `preview` image that people can use.  In theory (!). :smile:

**Comment by Avey777** [comment]
I just found that the Python backend cannot run properly after installing these packages locally. Since I am not very familiar with these dependencies, this is only provided as a reference for everyone.

**Comment by justinclift** [comment]
No worries.  Letting people know is a good idea. :smile:
